Jenkinsfile.cloud

```diff
@@ -105,19 +115,26 @@ node(env.NODE) {
     }
 
     stage("Generate Metadata") {
-        def prev_commit = sh(returnStdout: true,
-                             script: "ostree --repo=${repo} rev-parse ${commit}^").trim()
+        // we're stuck with an older host because imagefactory produces kernel
+        // panics on newer hosts, so let's use a container to have access to a
+        // newer rpm-ostree that understands the new pkglist metadata
+        docker.image(DOCKER_IMG).inside(DOCKER_ARGS) {
+            sh "dnf install -y rpm-ostree ostree"
```

Comment:
Nonblocking: After the composer container change lands, this could use that container.